gocourse02/zoo/interactionType.go

```diff
@@ -0,0 +1,21 @@
+package zoo
+
+type InteractionType int
+
+const (
+	Catch InteractionType = iota
+	Miss
+	LetGo
+)
+
+func (s InteractionType) String() string {
+	switch s {
+	case Catch:
+		return "Catch"
+	case Miss:
+		return "Miss"
+	case LetGo:
+		return "LetGo"
+	}
+	return "unknown"
```

Comment:
Знову ж таки, для консистенції, бо скрізь з великої:
```suggestion
	return "Unknown"
```